server/shared/src/main/scala/org/http4s/server/middleware/ResponseLogger.scala

```diff
@@ -36,6 +36,8 @@ import org.typelevel.ci.CIString
   */
 object ResponseLogger {
   private[this] val logger = Platform.loggerFactory.getLogger
+  private[this] val errorLogger =
+    Platform.loggerFactory.getLoggerFromName("org.http4s.server.service-errors")
```

Comment:
FWIW, we're already instantiating the logger with that name in `org.http4s.server`, so it should be cached already.